cmd/aries-js-worker/README.md

```diff
@@ -11,33 +11,147 @@ To build you will need:
 
 ## Usage
 
-### Node.js
+> Note: the API is in the very early stages of development and is still subject to a few changes.
+
+### Entrypoints
+
+`aries-js-worker` has several entrypoints tailored to the environment and needs:
+
+* `dist/node/aries.js`: for use in node.js
+* `dist/web/aries.js`: for use in the browser
+* `dist/rest/aries.js`: for use in any environment but relying on an external
+  [REST controller API server](https://github.com/hyperledger/aries-framework-go/blob/master/docs/rest/README.md)
+  instead of the bundled webassembly module.
+
+### Snippet
+
+**Example:** accept a did-exchange invitation:
+
+```js
+// in the browser
+
+const aries = await new Aries.Framework({
+    assetsPath: "/public/dist/assets",
+    "agent-default-label": "dem-js-agent",
+    "http-resolver-url": [],
+    "auto-accept": true,
+    "outbound-transport": ["ws", "http"],
+    "transport-return-route": "all",
+    "log-level": "debug"
+})
+
+// sample invitation
+const invitation = {
+    "@id":"4d26ad47-c71b-4e2e-9358-0a76f7fa77e4",
+    "@type":"https://didcomm.org/didexchange/1.0/invitation",
+    "label":"demo-js-agent",
+    "recipientKeys":["7rADm5sA9FHB4enuYXj6PJZDAm1JcesKmbtx7Qh8YZrg"],
+    "serviceEndpoint":"routing:endpoint"
+};
+
+// listen for connection 'received' notification
+aries.startNotifier(notice => {
+    const connection = notice.payload
+    // accept invitation
+    aries.didexchange.acceptInvitation(connection.connection_id)
+}, ["connections"])
+// receive invitation
+aries.didexchange.receiveInvitation(invitation)
+
+// listen for connection 'completed' notification
+aries.startNotifier(notice => {
+    const connection = notice.payload
+    if (connection.state === "completed") {
+        console.log("connection completed!")
+    }
+}, ["connections"])
+
+// release resources
+aries.destroy()
+```
+
+### Browser
+
+Note: make sure the assets are [served correctly](#important---serving-the-assets).
+
+Source `aries.js` in your `<script>` tag:
+
+```html
+<script src="dist/web/aries.js"></script>
+```
+
+Then initialize your aries instance:
+
+```js
+const aries = await new Aries.Framework({
+    assetsPath: "/path/serving/the/assets",
+    "agent-default-label": "dem-js-agent",
+    "http-resolver-url": [],
+    "auto-accept": true,
+    "outbound-transport": ["ws", "http"],
+    "transport-return-route": "all",
+    "log-level": "debug"
+})
+```
+
+### REST
 
-View [`App.js`](./App.js) for examples.
+Note: make sure the assets are [served correctly](#important---serving-the-assets) if you're running aries in the browser.
+
+Assuming you're in the browser, source `aries.js` in your `<script>` tag:
+
+```html
+<script src="dist/rest/aries.js"></script>
+```
+
+Then initialize your aries instance:
+
+```js
+const aries = await new Aries.Framework({
+    assetsPath: "/path/serving/the/assets", // still required for assets other than the wasm
```

Comment:
Do we have any doc on these startup params ? May be we can point to https://github.com/hyperledger/aries-framework-go/blob/master/docs/rest/agent_cli.md#agent-parameters, if these are similar.

Reply:
ignore - found it https://github.com/hyperledger/aries-framework-go/pull/1373/files#diff-a43de5ddb6dcf66800367a9232d43f90R84.